userguide/payment/includes/payment-states.adoc

```diff
@@ -55,29 +55,30 @@ Note that the first 3 cases are normal cases but the last 2 are errors that are
 
 If a payment is aborted by a control plugin, a payment attempt row is recorded with a state of ABORTED (no payment nor payment transaction is created). Kill Bill will return a 422 HTTP code.
 
+[[Janitor]]
 === Janitor
 
-PENDING and UNKNOWN transaction states should be temporary: the payment plugin needs to eventually tell Kill Bill what the final state of these transactions should be. This can be done explicitly by the plugin by calling the `notifyPendingTransactionOfStateChanged` PaymentApi (for instance, when receiving a webhook event from the gateway) or implicitly by the Janitor.
+PENDING and UNKNOWN transaction states should be temporary: the payment plugin needs to eventually tell Kill Bill what the final state of these transactions should be. This can be done explicitly by the plugin by calling the https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L402[`PaymentApi.notifyPendingTransactionOfStateChanged`] (for instance, when receiving a webhook event from the gateway) or implicitly by the Janitor.
```

Comment:
```suggestion
PENDING and UNKNOWN transaction states should be temporary; the payment plugin needs to eventually tell Kill Bill what the final state of these transactions should be. This can be done explicitly by the plugin by calling the https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L402[`PaymentApi.notifyPendingTransactionOfStateChanged`] (for instance, when receiving a webhook event from the gateway) or implicitly by the Janitor).
```